indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -480,9 +480,12 @@
   {:_id  {:path "concept-id"}}
   (merge
     {:concept-id (m/stored m/string-field-mapping)
+     :revision-id (m/stored m/int-field-mapping)
 
      :native-id (m/doc-values m/string-field-mapping)
      :native-id.lowercase (m/doc-values m/string-field-mapping)
+     :native-id-stored (-> m/string-field-mapping m/stored m/doc-values)
+     :deleted (m/stored m/bool-field-mapping) ; deleted=true is a tombstone
```

Comment:
We do not need this field.

Reply:
It can go both ways. This is to be consistent with other concept types even though it will always be false for granules.

Reply:
I am going to force push this branch with the update so that I can do the backport easier with just one commit.